anndata/dev/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .multi_files import AnnDataSet
```

Comment:
Both of the submodules should be private so that users do not see the lookups: `._multi_files` and `._pytorch`.